src/report/ReportManager.ts

```diff
@@ -102,17 +106,16 @@ export class ReportManager {
      * The following MUST hold true:
      * - the reporter's id is `reporterId`;
      * - the reporter is a member of `roomId`;
-     * - `eventId` did take place in room `roomId`;
-     * - the reporter could witness event `eventId` in room `roomId`;
+     * - `event` did take place in room `roomId`;
+     * - the reporter could witness event `event` in room `roomId`;
      * - the event being reported is `event`;
      *
      * @param roomId The room in which the abuse took place.
-     * @param eventId The ID of the event reported as abuse.
      * @param reporterId The user who reported the event.
      * @param event The event being reported.
      * @param reason A reason provided by the reporter.
      */
-    public async handleServerAbuseReport({ reporterId, event, reason }: { roomId: string, eventId: string, reporterId: string, event: any, reason?: string }) {
+    public async handleServerAbuseReport(roomId: string, reporterId: string, event: any, reason?: string) {
```

Comment:
Please don't change this API. The reason for which it is an object is because that's how we do named arguments in JavaScript.